website/cue/reference/components/sinks/base/socket.cue

```diff
@@ -453,9 +453,10 @@ base: components: sinks: socket: configuration: {
 		description: "The type of socket to use."
 		required:    true
 		type: string: enum: {
-			tcp:  "Send over TCP."
-			udp:  "Send over UDP."
-			unix: "Send over a Unix domain socket (UDS)."
+			tcp:           "Send over TCP."
+			udp:           "Send over UDP."
+			unix:          "Send over a Unix domain socket (UDS), in stream mode."
```

Comment:
Can we rename this `unix_stream` (while keeping an alias for `unix` for compatibility)? I'd like it to match what you see for the `socket` source: https://vector.dev/docs/reference/configuration/sources/socket/#mode